### PR TITLE
Feature/add user by email

### PR DESCRIPTION
**Pull request by Bloodyhacker**

### CODE REVIEW

**Comment by Bloodyhacker** [comment]
@geekbeast @makosblade

**Comment by Bloodyhacker** [comment]
Can anyone help me to merge this? @makosblade @geekbeast

File: src/main/java/com/kryptnostic/directory/v1/http/UserDirectoryApi.java [hunk 1]
```diff
@@ -137,5 +141,8 @@
 
     @GET( CONTROLLER + INITIALIZED + REALM_PATH )
     Iterable<UUID> listInitializedUserInRealm( @Path( REALM ) String realm ) throws BadRequestException;
+    
+    @GET( CONTROLLER + VALIDATE + SHARING + EMAIL_PATH )
```
Comment (makosblade) [inline src/main/java/com/kryptnostic/directory/v1/http/UserDirectoryApi.java:145]: Did this get updated in the code that implements this method?

Comment (makosblade) [inline src/main/java/com/kryptnostic/directory/v1/http/UserDirectoryApi.java:145]: lol, nevermind

**Comment by makosblade** [comment]
LGTM

**Pull request merged**